package/gluon-core/luasrc/lib/gluon/upgrade/115-swconfig

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/lua
+
+local platform = require 'gluon.platform'
+local uci = require('simple-uci').cursor()
+
+local function get_keys(t)
+	local keys={}
+	local n=0
+
+	for k,_ in pairs(t) do
+		n=n+1
+		keys[n]=k
+	end
+	return keys
```

Comment:
Consistent spacing for assignments.

Reply:
resolved in 7cab846e9ceed11755c29030864962bc345da2ef